src/components/OnboardingForm.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
```

Comment:
Why are you using `"use client";` here?

This component does not have states

Reply:
No special reason. We discussed this when coding and even though it works as an SSR component, for now, in the future as we use hooks for state management of the form itself, we're  likely making it a client-side component. But we don't need it for now, indeed